scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -850,6 +854,10 @@ def validate_model_spec(asset_config: assets.AssetConfig) -> int:
         _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.LICENSE} missing")
         error_count += 1
 
+    if not model.tags.get(MLFlowModelTags.HIDDEN_LAYERS_SCANNED):
+        _log_error(asset_config.file_name_with_path, f"{MLFlowModelTags.HIDDEN_LAYERS_SCANNED} missing")

```

Comment:
Can you please add text stating something like. "Model is not scanned by HiddenLayer ModelScanner tool. Please scan the model and retry". instead of tag missing. Folks will simply add the tag and submit.

Reply:
make sense

Reply:
i hope this is not skippable.

Reply:
updated